app/controllers/admin/users_controller.rb

```diff
@@ -342,15 +342,20 @@ def reject_bulk
   end
 
   def disable_second_factor
-    guardian.ensure_can_disable_second_factor! @user
-    if @user.user_second_factor.try(:delete)
```

Comment:
@featheredtoast Generally we want to use `destroy!` so that if it fails to destroy for whatever reason, it'll at least raise an error